.yalc/msupply-ble-service/package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "msupply-ble-service",
-  "version": "0.5.3",
+  "version": "0.5.1",
```

Comment:
why would the version decrease?

Reply:
I wonder if I branched off main but we build from develop or something. Will take a quick look at this tomorrow.

Reply:
It was this, now updated from main